### ISSUE TITLE
Error writing SVGs of simplified map

### ISSUE THREAD

**Issue by snaily** [mbloch/mapshaper#160]
I'm getting crashes at SVG export (in this case, starting from the Natural Earth 100m country shapefile), but only when I've done simplification. I'm guessing there's some null shapes around? The keep-shapes argument to -simplify makes no difference.

Mapshaper version: 0.3.41

```
$ mapshaper ne_110m_admin_0_countries/ne_110m_admin_0_countries.shp -filter-fields iso_a2 -filter '!["AQ", "GL"].includes(iso_a2)' -simplify 40% keep-shapes -proj +proj=vandg +lon_0=150 -info -o world.svg format=svg id-field=iso_a2
[filter] Retained 175 of 177 features
[simplify] Repaired 0 intersections; 481 intersections could not be repaired
Layer name: ne_110m_admin_0_countries
Records: 175
Geometry
  Type: polygon
  Bounds: -18581994.71904372 -8179425.299624527 19492917.245777547 15863742.913116159
  Proj.4: +proj=vandg +lon_0=150 +ellps=WGS84
Attribute data
  Field   First value
  iso_a2  'AF'
/usr/local/lib/node_modules/mapshaper/mapshaper.js:1574
    throw err;
    ^

TypeError: Cannot read property '1' of null
    at ArcCollection.forEachArcSegment (/usr/local/lib/node_modules/mapshaper/mapshaper.js:2429:28)
    at ArcCollection.forEachSegment (/usr/local/lib/node_modules/mapshaper/mapshaper.js:2442:21)
    at Object.MapShaper.getAvgSegment2 (/usr/local/lib/node_modules/mapshaper/mapshaper.js:2926:20)
    at Object.MapShaper.calcSegmentIntersectionStripeCount (/usr/local/lib/node_modules/mapshaper/mapshaper.js:4673:26)
    at Object.findSegmentIntersections (/usr/local/lib/node_modules/mapshaper/mapshaper.js:4584:33)
    at Object.MapShaper.findClippingPoints (/usr/local/lib/node_modules/mapshaper/mapshaper.js:5206:33)
    at Object.MapShaper.divideArcs (/usr/local/lib/node_modules/mapshaper/mapshaper.js:5071:26)
    at Object.MapShaper.addIntersectionCuts (/usr/local/lib/node_modules/mapshaper/mapshaper.js:5051:23)
    at Object.MapShaper.setCoordinatePrecision (/usr/local/lib/node_modules/mapshaper/mapshaper.js:12289:23)
    at Object.MapShaper.transformCoordsForSVG (/usr/local/lib/node_modules/mapshaper/mapshaper.js:12235:13)
```

**Comment by mbloch** [comment]
I just published a fix (v0.3.42). Thanks for this bug report :)

**Comment by mbloch** [comment]
Btw, the Natural Earth file you're using has some minor topology errors, which can be fixed by importing with the `snap` flag, like this:

`mapshaper ne_110m_admin_0_countries/ne_110m_admin_0_countries.shp snap`

The problem areas show up as red dots in the web interface:

![image](https://cloud.githubusercontent.com/assets/1232222/22075690/fd6f9f3a-dd7a-11e6-8da9-d745031c4fd7.png)
